_data/sidebar.yml

```diff
@@ -24,35 +24,35 @@ docs:
 
   - title: DOCUMENTS AND SEARCH DEFINITIONS
     documents:
-      - page: Documents and Document Ids
+      - page: Documents
         url: /documentation/documents.html
-      - page: Document Search Definitions
+      - page: Search Definitions
         url: /documentation/search-definitions.html
 
   - title: WRITING TO VESPA
     documents:
       - page: Writing to Vespa
         url: /documentation/writing-to-vespa.html
-      - page: Vespa Java Feeding Client
+      - page: Vespa HTTP Feeding Client
         url: /documentation/vespa-http-client.html
       - page: Feed using Hadoop, Pig, Oozie
         url: /documentation/feed-using-hadoop-pig-oozie.html
-      - page: Java Document API
+      - page: Document API
         url: /documentation/document-api-guide.html
       - page: Document Processing
         url: /documentation/document-processing-overview.html
-      - page: Document Annotations
+      - page: Annotations API
         url: /documentation/annotations.html
 
   - title: QUERYING VESPA
     documents:
       - page: Search API
         url: /documentation/search-api.html
-      - page: Vespa Query Language
+      - page: Query Language - YQL
```

Comment:
Maybe keep this as Vespa Query Language, but change the on-page topic to "Vespa Query Language - YQL"